torch/_inductor/codegen/cpp.py

```diff
@@ -2586,17 +2584,20 @@ def run(kernel):
             return
 
         def select_tiling_indices():
+            rw_index = []
             all_index = []
             for node in nodes:
-                rw = dependencies.extract_read_writes(node._body, *node._sizes)
-                all_index += [dep.index for dep in itertools.chain(rw.reads, rw.writes)]
+                rw_index += [*node._body.reads, *node._body.writes]
+                all_index += list(node._body.indexing_exprs.values())
```

Comment:
all_index has a somewhat confusing name, since it only includes index_exprs.

Reply:
update the variable name to avoid confusing.